app/screens/channel/channel_base.js

```diff
@@ -255,13 +257,24 @@ export default class ChannelBase extends PureComponent {
             theme,
         } = this.props;
 
+        const errorTitle = {
+            id: t('error.team_not_found.title'),
+            defaultMessage: 'Team Not Found',
+        };
+
+        const errorDescription = {
+            id: t('mobile.failed_network_action.shortDescription'),
+            defaultMessage: 'Make sure you have an active connection and try again.',
+        };
+
         if (!currentChannelId) {
             if (channelsRequestFailed) {
-                const PostListRetry = require('app/components/post_list_retry').default;
                 return (
-                    <PostListRetry
-                        retry={this.retryLoadChannels}
+                    <FailedNetworkAction
```

Comment:
we need to Lazy load this with an inline require as the previous `PostListRetry` that we had